mod_reforged/config/perk_defs.nut

```diff
@@ -127,6 +127,14 @@
 		Icon = "ui/perks/rf_small_target.png",
 		IconDisabled = "ui/perks/rf_small_target_bw.png",
 	},
+	{
+		ID = "perk.rf_second_wind",
+		Script = "scripts/skills/perks/perk_rf_second_wind",
+		Name = ::Const.Strings.PerkName.RF_SecondWind,
+		Tooltip = ::Const.Strings.PerkDescription.RF_SecondWind,
+		Icon = "ui/perks/rf_second_wind.png",
+		IconDisabled = "ui/perks/rf_second_wind_bw.png",
```

Comment:
We decided to use `_sw` for all perk icons. The `_bw` is from older icons before we decided that and will be eventually changed to `_sw`.